cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package encryptionconfig
+// Package encryption handles assigning encryption providers to each group resource storage.
+package encryption
```

Comment:
Specifically, the cloud provider stuff I wanted out. I also think it's bad form to mix up the config file parsing with the construction of individual translators, as the config file isn't reusable (if, say, we want to add this external call-out) but the other code is. @liggitt do you know of actual reuse of this code, or is this just a theoretical concern?